nomad/acl_endpoint.go

```diff
@@ -263,6 +264,14 @@ func (a *ACL) GetPolicy(args *structs.ACLPolicySpecificRequest, reply *structs.S
 			reply.Policy = out
 			if out != nil {
 				reply.Index = out.ModifyIndex
+
+				rules, err := policy.Parse(out.Rules)
+
+				if err != nil {
+					return err
+				} else {
```

Comment:
No `else {` needed here. The Go idiom for the `if err` boilerplate is to just do an early return and not worry about nesting subsequent code in an else block. nbd

Reply:
I removed it, thanks.